### PR TITLE
feat(service): added reverse proxy service to cloud toolkit

### PR DESCRIPTION
**Pull request by madhavilosetty-intel**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -27,8 +27,8 @@ services:
     env_file:
       - .env
     environment: 
-      RPS_USER: ${RPS_MPS_USER}
```
Comment (rsdmike) [inline docker-compose.yml:30]: 👍🏻

File: docker-compose.yml [hunk 2]
```diff
@@ -44,8 +44,20 @@ services:
...
       MPS_PASS: ${MPS_PASSWORD}
+      MPS_INSTANCE_NAME: '{{.Task.Name}}'
     volumes:
       - private-volume:/mps-microservice/private
+  mpslookup:
+    image: mpslookup:latest
+    environment:
+      MPS_CONNECTION_STRING: postgresql://postgresadmin:admin123@db:5432/mpsdb?sslmode=disable
+    ports:
+      - "8003:8003"
```
Comment (rsdmike) [inline docker-compose.yml:55]: I don't think we need to keep this exposed do we?

File: kong.yaml [hunk 3]
```diff
@@ -25,8 +25,8 @@ _transform: true
 
 services:
 - name: mps-rest
-  host: mps
-  port: 3000
+  host: mpslookup
```
Comment (rsdmike) [inline kong.yaml:28]: We'll need to figure out the best way to do this---this is only for swarm , it is not needed for compose-- but maybe we just always have it...thoughts?

File: docker-compose.yml (hunk 2 above)
Comment (madhavilosetty-intel) [reply docker-compose.yml:55]: done

File: kong.yaml (hunk 3 above)
Comment (madhavilosetty-intel) [reply kong.yaml:28]: Can I revert the changes back to mps and 3000 port? Document the steps to modify the kong.yaml, if it needs to deploy in docker swarm.

**Review by rsdmike — APPROVED** [review]

**Pull request merged**